### ISSUE TITLE
Assertion violation at src/tsolvers/lasolver/Simplex.cc:227

### ISSUE THREAD

**Issue by rainoftime** [usi-verification-and-security/opensmt#83]
Hi, for the following formula,
~~~~
(set-logic QF_LIA)
(declare-fun i2 () Int)
(declare-fun i4 () Int)
(declare-fun i5 () Int)
(assert (distinct i2 858))
(assert (< i5 i4))
(push 1)
(assert (= (* (- i2 i5) 7) 1000))
(push 1)
(assert false)
(check-sat)
(assert true)
(check-sat)
(assert false)
(check-sat)
(pop 1)
(check-sat)
~~~~

opensmt (commit 04c7fa3) throws an assertion violation
~~~~
; this binary is compiled in debug mode (slow)
; git hash: e9e14b4bc1cae5fb95a34ce2108642cc6d6ef068
unsat
unsat
unsat
opensmt: /home/opensmt/src/tsolvers/lasolver/Simplex.cc:227: Simplex::Explanation Simplex::assertBoundOnVar(LVRef, LABoundRef): Assertion `!model->isUnbounded(it)' failed.
Aborted (core dumped)
~~~~

**Comment by blishko** [comment]
This does not crash anymore, but it seems OpenSMT goes into infinite branching in LIA solver.
We should look into that issue.

**Comment by blishko** [comment]
Fixed by commit 68e3a29f in #111
